src/python_testing/TC_OPCREDS_3_2.py

```diff
@@ -0,0 +1,184 @@
+#
```

Comment:
can you please add this to the .github/workflows/tests.yaml file so it gets run in the CI?

Reply:
Done